fe/fe-core/src/main/java/com/starrocks/backup/BackupJobInfo.java

```diff
@@ -449,8 +457,19 @@ private static void genFromJson(String json, BackupJobInfo jobInfo) {
             } catch (Exception e) {
                 tblInfo.autoIncrementId = null;
             }
+            try {
+                tblInfo.tableType = tbl.getInt("tableType");
+            } catch (Exception e) {
+                tblInfo.tableType = Table.TableType.OLAP.ordinal();
+            }
             JSONObject parts = tbl.getJSONObject("partitions");
             String[] partsNames = JSONObject.getNames(parts);
+            Table.TableType tblType = Table.TableType.values()[tblInfo.tableType];
```

Comment:
What if no tableType in the table info in old version ?

Reply:
Updated, if restore older version of snapshot, treat the type as Table.TableType.OLAP is fine.